docs/AddressSanitizing.md

```diff
@@ -0,0 +1,31 @@
+# Address Sanitizing
+
+*AddressSanitizer* (aka ASan) is a very useful library for detecting memory-related issues for C/C++. It consists of a compiler instrumentation module and a run-time library. The typical performance penalty for address sanitizing is 2x.
+
+This file details the how *Address Sanitization* is performed within the  `ebpf-for-windows.sln` solution and its CI/CD pipeline.
+
+
+## Current usage in eBPF for Windows
+
+*AddressSanitizer* is integrated with the Visual Studio project system, and is installed as a component through the *Visual Studio Installer*. Address sanitization can be enabled on a C/C++ project by adding `/fsanitize=address` to the C/C++ compiling options (either through the command line or the dedicated UI field).
+
+On Windows, address sanitization must be implemented differently for kernel-mode and user-mode modules:
```

Comment:
Why must it be different? I think the "must" doesn't add much value here. Maybe "On Windows address sanitization is implemented differently in user mode and kernel mode".